llvm/lib/Transforms/Utils/LoopRotationUtils.cpp

```diff
@@ -551,7 +601,15 @@ bool LoopRotate::rotateLoop(Loop *L, bool SimplifiedLatch) {
       if (L->hasLoopInvariantOperands(Inst) && !Inst->mayReadFromMemory() &&
           !Inst->mayWriteToMemory() && !Inst->isTerminator() &&
           !isa<DbgInfoIntrinsic>(Inst) && !isa<AllocaInst>(Inst)) {
+
+        auto DbgValueRange = LoopEntryBranch->cloneDebugInfoFrom(Inst, NextDbgInst);
+        // Remap any new instructions,
```

Comment:
Unfinished comment?

Reply:
Removed, and shuffled the 'clone' into the guard. I think it should be self explanatory.